packages/mint-components/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@saasquatch/mint-components",
-  "version": "1.4.2",
+  "version": "1.4.3-0",
```

Needs to be based on production

```suggestion
  "version": "1.5.1",
```